### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by namishkhanna**
**Resolved the problem of opening PDF in Same tab to New tab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ If you are using EvalAI for hosting challenges, please cite the following techni
 }
 ```
 <p>
-    <a href="http://learningsys.org/sosp19/assets/papers/23_CameraReadySubmission_EvalAI_SOSP_2019%20(8)%20(1).pdf"><img src="docs/source/_static/img/evalai-paper.jpg"/></a>
+    <a href="https://namishkhanna.github.io/newtab/?url=http://learningsys.org/sosp19/assets/papers/23_CameraReadySubmission_EvalAI_SOSP_2019%20(8)%20(1).pdf"><img src="docs/source/_static/img/evalai-paper.jpg"/></a>
```
Comment (RishabhJain2018) [inline README.md:89]: Can we please use `target="_blank"` in the anchor tag :)

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (namishkhanna) [reply README.md:89]: @RishabhJain2018 Sorry, but we can't as in readme.md File Github doesn't have feature to open New Tab

Comment (namishkhanna) [reply README.md:89]: @RishabhJain2018 Please tell me what to do ?

Comment (RishabhJain2018) [reply README.md:89]: I see, then I would prefer to keep it as it is. Using `https://namishkhanna.github.io/newtab/?url` seems like a hack to me.

Thank you for the effort. Please feel free to take up any other issue.

**Pull request closed without merging**